pkg/steps/artifacts.go

```diff
@@ -505,3 +511,65 @@ func allPodContainerNamesWithArtifacts(pod map[string]interface{}) map[string]st
 	}
 	return names
 }
+
+func gatherContainerLogsOutput(podClient PodClient, artifactDir, namespace, podName string) error {
+	var validationErrors []error
+	list, err := podClient.Pods(namespace).List(meta.ListOptions{FieldSelector: fields.Set{"metadata.name": podName}.AsSelector().String()})
+	if err != nil {
+		return fmt.Errorf("could not list pod: %v", err)
+	}
+	pod := &list.Items[0]
+
+	if pod.Annotations["ci-operator.openshift.io/containers-logged-on-failure"] == "" {
```

Comment:
I think the case was made earlier that this could be "container-logs-as-artifacts" (don't have to deal with failure).  That's simpler, and getting the extra logs isn't that bad